### PR TITLE
Add subsection "Migration of Forking Market"

### PR DESCRIPTION
**Pull request by CrystalBallBe**
I added the subsection "Migration of Forking Market" to "Migration of the objects".
@MicahZoltu Could you review this PR?

### CODE REVIEW

**Comment by CrystalBallBe** [comment]
Demo: https://crystalballbe.github.io/version2-english/9-fork/4-migration-of-the-objects.html

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 1]
```diff
@@ -70,6 +75,44 @@ This can be summarized as follows:
 # Child universe is created by REP holders
 A [child universe]({{glossary_child_universe}}) is created when the [REP]({{glossary_reputation_token}}) holder who *first* migrates their REP to it. Even if the [forking market]({{glossary_forking_market}}) has [finalized]({{glossary_finalized_market}}) or the [forking period]({{glossary_forking_period}}) ends, the [child universe]({{glossary_child_universe}}) is not created without migrating REP. And the REP holder who *first* migrates their REP has to pay not only the [transaction fee]({{glossary_transaction_fee}}) of migrating their REP, but also the transaction fee of creating the child universe.
 
+# Migration of Forking Market
+## Market Itself
+The [forking market]({{glossary_forking_market}}) itself is migrated to each [child universe]({{glossary_child_universe}}). In a child universe, the forking market is [finalized]({{glossary_finalized_market}}) for each possible [outcome]({{glossary_outcome}}) of it. (See [basic knowledge]({{url_basic_knowledge}}#what-happens-when-a-fork-starts) for details on how child universes are created.)
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
The [forking market]({{glossary_forking_market}}) is duplicated in each [child universe]({{glossary_child_universe}}). In a child universe, the forking market is [finalized]({{glossary_finalized_market}}) as the [outcome]({{glossary_outcome}}) that aligns with that particular child universe. (See [basic knowledge]({{url_basic_knowledge}}#what-happens-when-a-fork-starts) for details on how child universes are created.)
```

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 2]
```diff
@@ -70,6 +75,44 @@ This can be summarized as follows:
...
+
+The forking market is migrated to the child universe when [REP]({{glossary_reputation_token}}) holder *first* migrates their REP to the child universe. In other words, the forking market is not migrated until REP is migrated to the child universe by REP holder.
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
This duplication occurs when the child universe is created.  This happens when the *first* [REP]({{glossary_reputation_token}}) holder migrates their REP to that child universe.
```

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 3]
```diff
@@ -70,6 +75,44 @@ This can be summarized as follows:
...
+
+## Staked REP
+The staked [REP]({{glossary_reputation_token}}) on any [outcome]({{glossary_outcome}}) of the [forking market]({{glossary_forking_market}}) is migrated to the [child universe]({{glossary_child_universe}}) corresponding to the staked outcome.
+
+Those REP will be unstaked and refunded to their owners in a child universe when the migration has done. And if the child universe to which the REP are migrated is the [winning universe]({{glossary_winning_universe}}), its owners receive a 40% [ROI]({{glossary_roi}}) on their stake in the winning universe.
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
That REP will be unstaked and refunded to their owners in a child universe when the migration has done. They will also receive a 40% [ROI]({{glossary_roi}}) in additional REP minted in the universe they staked on prior to the fork.
```
The 40% ROI goes to *anyone* who staked in disputes leading up to the fork, not only the winning universe.

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 4]
```diff
@@ -70,6 +75,44 @@ This can be summarized as follows:
...
+
+## Attached Objects
+The objects which are attached to the [forking market]({{glossary_forking_market}}) are:
+ - [share]({{glossary_share}})
+ - [unfilled order]({{glossary_unfilled_order}})
+ - [creator fee]({{glossary_creator_fee}})
+ - [validity bond]({{glossary_validity_bond}})
+
+When the forking market is [finalized]({{glossary_finalized_market}}), those objects are migrated to the [winning universe]({{glossary_winning_universe}}) by the Augur contracts automatically.
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
When the for is [finalized]({{glossary_finalized_market}}), those objects are migrated to the [winning universe]({{glossary_winning_universe}}) by the Augur contracts automatically.
```

**Review by MicahZoltu — APPROVED** [review]
A few minor points and corrections.  I changed "migrated" to "duplicated" in the forking market descriptions since "migrated" implies a single destination which isn't quite right.

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md (hunk 3 above)
Comment (CrystalBallBe) [reply collections/_version2-english/9-fork/4-migration-of-the-objects.md]: > The 40% ROI goes to anyone who staked in disputes leading up to the fork, not only the winning universe.

Even REP that migrated to the *losing* universe will receive 40% ROI if the REP staked in disputes leading up to the fork. I finally understand.

I remember that you told me the same thing before, but I didn't understand it correctly at that time. I will check if there are any mistakes in the sentences I merged in the past. Thanks!

FWIW, I tried to verify that and might find the applicable line in the contracts. It appears to be minted +40% REP for all created child universes without distinction between *winning* and *losing*. https://github.com/AugurProject/augur/blob/ead8260305ed9ef83f094c3d7e9b5013f9c57680/packages/augur-core/src/contracts/reporting/BaseReportingParticipant.sol#L30

**Pull request merged**